database/tag.js

```diff
@@ -1,15 +1,18 @@
 var Sequelize = require('sequelize');
 
 module.exports = function(db) {
-  var Projects = db.define('projects', {
+  var Tag = db.define('Tag', {
     id: {
-      type: Sequelize.STRING,
-      unique: 'projectid',
+      type: Sequelize.UUID,
       allowNull: false,
```

Comment:
If we are using UUID fields and we want the database to auto-generate them, we should be able to use something like:

`defaultValue: Sequelize.UUIDV4` as per http://docs.sequelizejs.com/variable/index.html